pkgs/tools/audio/mpris-scrobbler/default.nix

```diff
@@ -15,15 +15,19 @@
 
 stdenv.mkDerivation rec {
   pname = "mpris-scrobbler";
-  version = "0.4.95";
+  version = "0.5.0";
 
   src = fetchFromGitHub {
     owner  = "mariusor";
     repo   = "mpris-scrobbler";
     rev    = "v${version}";
-    sha256 = "sha256-Cqf0egS4XSDiKLdizNHPdS0Zt3jQxw9e78S7n23CuKI=";
+    sha256 = "sha256-HUEUkVL5d6FD698k8iSCJMNeSo8vGJCsExJW/E0EWpQ=";
   };
 
+  mesonFlags = [
+    "-Dwerror=false" "-Dversion=${version}"
```

Comment:
Weird, it looks to me like `-Werror` has actually been removed: https://github.com/mariusor/mpris-scrobbler/compare/v0.4.95...v0.5.0
What is the error message? Maybe report it upstream?

Also it might be cleaner to have one flag per line to make `git blame` in the future work better.